Assets/PatchKit/UnityEditorExtension/Editor/UI/SafeBuildAndUploadScreen.cs

```diff
@@ -247,6 +247,13 @@ private void DrawVersion()
                 EditorGUILayout.Toggle(_overwriteDraftVersion);
         }
         EditorGUILayout.EndHorizontal();
+        EditorGUILayout.BeginHorizontal();
+        {
+            EditorGUILayout.LabelField("Remove files .pdb after build");

```

Comment:
Change to "Include *.pdb files". It should be enabled by default. The variable may stay as it is, but you need to negate it.

Reply:
done